firestore/objc/firestore-smoketest-objc/ViewController.m

```diff
@@ -80,7 +80,8 @@ - (void)viewDidLoad {
 - (void)setupCacheSize {
     // [START fs_setup_cache]
     FIRFirestoreSettings *settings = [FIRFirestore firestore].settings;
-    settings.cacheSizeBytes = kFIRFirestoreCacheSizeUnlimited;
+    settings.cacheSettings =
+        [[FIRPersistentCacheSettings alloc] initWithSizeBytes:@1000000];
```

Comment:
If we are not using `kFIRFirestoreCacheSizeUnlimited` anymore, can we add a comment to explain what size is this in MB?

Reply:
Applies elsewhere.

Reply:
Done.